### DISCUSSION TITLE
Unlock/lock timeline layer with one click/touch

### DISCUSSION THREAD

**Discussion by kevincreative** [keiwando/toonsquid#474] [Ideas]
Currently unlocking a layer in the timeline is a 2 step process. Click/touch the lock icon and click/touch "unlock" in the list that opens up. It would be great if similar to the eye icon that works with one click/touch, users could just click/touch the lock icon once to unlock or lock the layer without opening the options list.

**Comment by ProductDesignMarco** [comment]
That would be great, we overall need more one-click actions with its corresponding keyboard shortcut.
